src/CMakeLists.txt

```diff
@@ -85,7 +85,6 @@ set(cryptoms_lib_files
     solverconf.cpp
     distillerlong.cpp
     distillerlongwithimpl.cpp
-    str_impl_w_impl.cpp
```

Comment:
Why was this removed?

Reply:
The reason is that when I first run the solver that is in the for-gergo branch it threw an error, which reason was that the mentiond file is not found. Since you wrote that you cleand the solver for this branch, I thought you deleted this file intentionally and forget to delete that reference, so in order to be able to run it I deleted this and the code where this was used.

So my question is that since as far as I can see it the exact file is not in the main branch, what to do with it?